src/codegen/constants.rs

```diff
@@ -1,28 +1,25 @@
 use std::path::Path;
 
 use crate::{
-    analysis::imports::Imports,
     codegen::general::{
         self, cfg_condition, cfg_deprecated, doc_alias, version_condition, version_condition_string,
     },
     env::Env,
     file_saver, library,
+    nameutil::use_glib_type,
 };
 
 pub fn generate(env: &Env, root_path: &Path, mod_rs: &mut Vec<String>) {
     let path = root_path.join("constants.rs");
-    let mut imports = Imports::new(&env.library);
 
     if env.analysis.constants.is_empty() {
         return;
     }
 
     let sys_crate_name = env.main_sys_crate_name();
-    imports.add("std::ffi::CStr");
```

Comment:
I would keep this and import GStr here, that would drop various lines of code.

Reply:
Sure, done :)